go/server.go

```diff
@@ -23,37 +24,29 @@ func init() {
 var (
 	PLAID_CLIENT_ID     = os.Getenv("PLAID_CLIENT_ID")
 	PLAID_SECRET        = os.Getenv("PLAID_SECRET")
-	PLAID_PUBLIC_KEY    = os.Getenv("PLAID_PUBLIC_KEY")
 	PLAID_PRODUCTS      = os.Getenv("PLAID_PRODUCTS")
 	PLAID_COUNTRY_CODES = os.Getenv("PLAID_COUNTRY_CODES")
 	// Parameters used for the OAuth redirect Link flow.
 	//
-	// Set PLAID_OAUTH_REDIRECT_URI to 'http://localhost:8000/oauth-response.html'
+	// Set PLAID_REDIRECT_URI to 'http://localhost:8000/oauth-response.html'
 	// The OAuth redirect flow requires an endpoint on the developer's website
 	// that the bank website should redirect to. You will need to configure
 	// this redirect URI for your client ID through the Plaid developer dashboard
 	// at https://dashboard.plaid.com/team/api.
-	PLAID_OAUTH_REDIRECT_URI = os.Getenv("PLAID_OAUTH_REDIRECT_URI")
-	// Set PLAID_OAUTH_NONCE to a unique identifier such as a UUID for each Link
-	// session. The nonce will be used to re-open Link upon completion of the OAuth
-	// redirect. The nonce must be at least 16 characters long.
-	PLAID_OAUTH_NONCE = os.Getenv("PLAID_OAUTH_NONCE")
+	PLAID_REDIRECT_URI = os.Getenv("PLAID_REDIRECT_URI")
 
 	// Use 'sandbox' to test with fake credentials in Plaid's Sandbox environment
 	// Use `development` to test with real credentials while developing
 	// Use `production` to go live with real users
 	APP_PORT = os.Getenv("APP_PORT")
 )
 
-var clientOptions = plaid.ClientOptions{
+var client, _ = plaid.NewClient(plaid.ClientOptions{
```

Comment:
nit / question: should we either:
- document that `plaid.NewClient` is never expected to return errors and we just haven't updated the clib yet?
- handle the error properly and preserve the option to return errors from `plaid.NewClient` in the future

Reply:
ill do the latter that's a good idea